### PR TITLE
Add Python 3.6 and drop Python 3.3

### PR DESCRIPTION
**Pull request by jamadden**
- Badges
- DRY for deps
- Enable coveralls and a coverage environment.
- 100% coverage

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,24 +1,21 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,pypy
+    py27,py34,py35,py36,pypy,pypy3
```
Comment (icemac) [inline tox.ini]: I do not like that the `coverage` environment is missing here.
But thank you for defining the `coverage` environment. This is better than not having it.

**Comment by icemac** [comment]
BTW I added you as owner on PyPI.

File: tox.ini (hunk 1 above)
Comment (jamadden) [inline tox.ini]: Added.

**Comment by jamadden** [comment]
Thanks! I've released 4.1.0 to PyPI.

**Pull request merged**